shared/hack/install.sh

```diff
@@ -171,6 +174,21 @@ install_grpc_cli() {
     fi
 }
 
+install_hypershift() {
+    # Build from code
```

Comment:
This should checkout a specific tag set in `shared/config/dependencies.sh`

Reply:
hyperhsift qe told me that we should take the latest one from main branch and I looked at the tag, the latest one  `2.0.0-20220406093220`  is too old.  I will put main as the version for now.

Reply:
Let's use `main` as the version number for the moment then.